docs/schema/merging.md

```diff
@@ -10,71 +10,6 @@ An OCDS [record](../getting_started/releases_and_records) aggregates all the rel
 * A compiled release is created by taking only the most recent values of fields from releases in a given contracting process.
 * A versioned release is created by taking all values of fields from releases in a given contracting process, copying metadata about the release from which they are taken, and putting them in chronological order.
 
-<div class="example hint" markdown=1>
```

Comment:
Please restore the old example and create an issue to remove it. Files under `schema/` can only be changed in new releases, whereas we can change guidance anytime between releases.